packages/backup/composer.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
It's really cool we're considering to add this functionality as a new package. I, however, was wondering if there was any special purpose behind that - are we planning to use this package anywhere else?

Reply:
I'm not sure if this will be useful in other projects down the line, but keeping it in a package: 
1) Keeps it contained in a logical manner, and
2) Would make it easier to split this out into a separate plugin, if we decide to down the line (as this is something we've discussed as a possibility).

Reply:
Sounds like a great reason to keep it in a package already, thanks for elaborating! 😉